Dockerfile

```diff
@@ -3,11 +3,17 @@ FROM wordpress:latest
 ARG woocommerce_version
 
 RUN apt-get update \
-    && apt-get install -y --no-install-recommends unzip wget gettext \
-    && wget https://downloads.wordpress.org/plugin/woocommerce.${woocommerce_version}.zip -O /tmp/temp.zip \
+    && apt-get install -y --no-install-recommends unzip wget gettext vim \
+    && wget https://downloads.wordpress.org/plugin/woocommerce.${woocommerce_version}.zip -O /tmp/woocommerce.zip \
+    && wget https://downloads.wordpress.org/plugin/relative-url.0.1.7.zip -O /tmp/relative-url.zip \
```

Comment:
The relative-url plugin is required for [ngrok integration](https://ngrok.com/docs#wordpress)